client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -57,6 +57,11 @@ export default React.createClass( {
 
 	monthlyPrice: function() {
 		const { cost, currency } = this.props.cartItem;
+
+		if ( typeof cost === 'undefined' ) {
+			return this.translate( 'Loading price' );
```

Comment:
In that case I'd rather add return no-break space or something like that to match the height. The redundancy seems weird.